### PR TITLE
Translated 'how-to-setup-freecodecamp-locally.md' into Korean

### PR DESCRIPTION
**Pull request by yoonboyeong**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
@nancydocode Please revert the change which deletes the `curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/nesting-for-loops.english.md` file from the repo.

**Comment by nancydocode** [comment]
@RandellDawson I am sorry. I deleted it because the changes in that file should not be merged into the original file and I found out it on this PR.

**Comment by RandellDawson** [comment]
Well, you need to remove the commit which deleted the file, because if we merge the PR, we would delete the file from the repo.

**Comment by nancydocode** [comment]
@RandellDawson I reverted the document. Would this way work? I am sorry to bother you.

**Comment by RandellDawson** [comment]
Looks fine now.

File: docs/i18n-languages/korean/how-to-setup-freecodecamp-locally.md [hunk 1]
```diff
@@ -0,0 +1,542 @@
+<!-- do not translate this -->
+| [Read these guidelines in other languages](/docs/i18n-languages) |
+|-|
+<!-- do not translate this -->
+
+# freeCodeCamp를 로컬 시스템에 설치하기
+로컬 시스템에(역주: 예를 들어서, 내 컴퓨터) freeCodeCamp를 설치하려고 하신다면 이 가이드라인을 따라 주세요. 주기적으로 기여(역주: 참여)해 주실 생각이라면 이를 추천드립니다.
+
+예를 들어서 가이드나 코딩챌린지 페이지 미리 보기라든지 코드를 디버깅하거나 버그를 고치는 등의 몇 가지 기여 흐름도는 여러분에게 freeCodeCamp를 로컬하게 진행하기를 기대합니다.
```
Comment (Violet-Bora-Lee) [inline docs/i18n-languages/korean/how-to-setup-freecodecamp-locally.md]: 몇 가지 기여 흐름도는~~~ 기대합니다.

주술 구조가 맞지 않습니다.

File: docs/i18n-languages/korean/how-to-setup-freecodecamp-locally.md [hunk 2]
```diff
@@ -0,0 +1,542 @@
...
+
+## GitHub에 있는 저장소를 복사하기
+['Forking'](https://help.github.com/articles/about-forks/)은 여러분만의 GitHubd에 있는 freeCodeCamp의 주요 저장소의 복사본을 갖게 해 줍니다.
```
Comment (Violet-Bora-Lee) [inline docs/i18n-languages/korean/how-to-setup-freecodecamp-locally.md]: GitHub오타 확인해주세요

File: docs/i18n-languages/korean/how-to-setup-freecodecamp-locally.md (hunk 1 above)
Comment (Violet-Bora-Lee) [inline docs/i18n-languages/korean/how-to-setup-freecodecamp-locally.md]: 몇 가지 기여 흐름도는~~~ 기대합니다.

주술 구조가 맞지 않습니다.

File: docs/i18n-languages/korean/how-to-setup-freecodecamp-locally.md [hunk 3]
```diff
@@ -0,0 +1,542 @@
+<!-- do not translate this -->
+| [Read these guidelines in other languages](/docs/i18n-languages) |
+|-|
+<!-- do not translate this -->
+
+# freeCodeCamp를 로컬 시스템에 설치하기
+로컬 시스템에(역주: 예를 들어서, 내 컴퓨터) freeCodeCamp를 설치하려고 하신다면 이 가이드라인을 따라 주세요. 주기적으로 기여(역주: 참여)해 주실 생각이라면 이를 추천드립니다.
```
Comment (Violet-Bora-Lee) [inline docs/i18n-languages/korean/how-to-setup-freecodecamp-locally.md]: - 역주는 옛날 표현방식이라 요즘 출판업계에선 옮긴이라는 표현을 많이 사용합니다.
- 역주는 되도록이면 지양하는것이 좋습니다. 두 역주를 제외해도 충분히 이해가 가는 문장인것 같습니다.

File: docs/i18n-languages/korean/how-to-setup-freecodecamp-locally.md [hunk 4]
```diff
@@ -0,0 +1,542 @@
...
+
+예를 들어서 가이드나 코딩챌린지 페이지 미리 보기라든지 코드를 디버깅하거나 버그를 고치는 등의 몇 가지 기여 흐름도는 여러분에게 freeCodeCamp를 로컬하게 진행하기를 기대합니다.
+
+## GitHub에 있는 저장소를 복사하기
+['Forking'](https://help.github.com/articles/about-forks/)은 여러분만의 GitHubd에 있는 freeCodeCamp의 주요 저장소의 복사본을 갖게 해 줍니다.
+
+이는 아주 중요합니다. GitHub에 있는 freeCodeCamp의 복사본을 갖게 해 주거나 저장소를 다운받아 로컬하게 작업하도록 도와주기 때문입니다. 후에, 여러분이 로컬하게 작업한 결과물을 fork를 통해 여러분이 가지고 있는 복사본에서메인 저장소로 옮기는 것을 pull request를 통해 요청할 수 있습니다.
```
Comment (Violet-Bora-Lee) [inline docs/i18n-languages/korean/how-to-setup-freecodecamp-locally.md]: 복사본에서메인

띄어쓰기 확인해주세요

**Review by Violet-Bora-Lee — CHANGES_REQUESTED** [review]
맞춤법과 주술 구조가 맞지않는 문장들이 있습니다.

http://speller.cs.pusan.ac.kr/

맞춤법 검사기 돌리셔서 커밋 다시 해주시면 리뷰 재진행하겠습니다.

File: docs/i18n-languages/korean/how-to-setup-freecodecamp-locally.md (hunk 1 above)
Comment (nancydocode) [reply docs/i18n-languages/korean/how-to-setup-freecodecamp-locally.md]: 어떻게 하면 좋을까요?

**Pull request merged**